src/app/services/modal.service.ts

```diff
@@ -25,20 +25,13 @@ export class ModalService {
     message: string,
     btnOkText: string = this.btnOkText,
     btnCancelText: string = this.btnCancelText,
-    dialogSize: 'sm'|'lg' = 'sm'): Promise<boolean> {
+    dialogSize: 'sm'|'lg' = 'sm'): Promise<any> {
     const modalRef = this.modalService.open(ModalComponent, { size: dialogSize, windowClass: 'modal-window' });
     modalRef.componentInstance.title = title;
     modalRef.componentInstance.message = message;
     modalRef.componentInstance.btnOkText = btnOkText;
     modalRef.componentInstance.btnCancelText = btnCancelText;
 
-    let result;
-    try {
-      result = await modalRef.result;
-    } catch {
-      result = false;
-    }
-
-    return result;
+    return modalRef.result;
```

Comment:
Je bent nog een 'oude' notatie vergeten te vervangen op: https://github.com/kadaster-labs/sensrnet-registry-frontend/blob/remove-organization/src/app/components/map/map.component.ts#L580 (ik link 'm hier, doordat die niet is gewijzigd kan ik daar geen comment plaatsen 😉)

Maar ik heb ik nog wel een vraag over. Was er een probleem met de vorige opzet? Er zat namelijk een rationale achter. De reden voor de oude opzet was dat `confirm` anders een error throws wanneer de popup dismissed wordt door buiten de popup te klikken. Die werd hier afgevangen, waardoor er altijd alleen een resultaat (accepted/`true` of rejected/`false`) werd gePromise'd. Nu moeten we er elke keer om denken een error callback toe te voegen om error logging te voorkomen bij popup dismissals. 😊

Reply:
Scherp gevonden! ... en fixed